codecov.yml

```diff
@@ -0,0 +1,13 @@
+# Customizations to codecov for c-t-go repo. This will be merged into
+# the team / default codecov yaml file.
+#
+# Validate changes with:
+# curl --data-binary @codecov.yml https://codecov.io/validate
+
+# Exclude code that's for testing, demos or utilities that aren't really
+# part of production releases.
+ignore:
+  - "trillian/integration/**/*"
+  - "trillian/mockclient/**/*"
```

Comment:
I think "mock_*.go" would be the better version of this, since it'll match all mocks, current and future, rather than just this one.

Reply:
Done.